tests/antithesis/Makefile

```diff
@@ -41,5 +44,38 @@ antithesis-run-local-validation:
 
 .PHONY: antithesis-clean
 antithesis-clean:
-	export USER_ID=$(USER_ID) && export GROUP_ID=$(GROUP_ID) && docker-compose down
+	export USER_ID=$(USER_ID); export GROUP_ID=$(GROUP_ID); docker-compose down
 	rm -rf /tmp/etcddata0 /tmp/etcddata1 /tmp/etcddata2 /tmp/etcdreport
+
+.PHONY: set-version-3.4
+set-version-3.4:
+	$(eval VERSION=3.4)
+	$(eval DOCKERFILE=./Dockerfile-release)
+
+.PHONY: antithesis-build-etcd-image-3.4
+antithesis-build-etcd-image-3.4: set-version-3.4 antithesis-build-etcd-image
```

Comment:
Please keep similar targets next to each other like `antithesis-build-etcd-image`, `antithesis-build-etcd-image-3.4` and `antithesis-build-etcd-image-3.5`